### PR TITLE
fix: Fix to pass mypy type-checking

### PR DESCRIPTION
**Pull request by jtran**

### CODE REVIEW

File: entrypoint.py [hunk 1]
```diff
@@ -58,12 +58,12 @@ def fetch_price_for_part(part_number: str) -> dict[int, float]:
             "X-API-KEY": api_key,
             "X-CLIENT-ID": client_id,
         },
-        params={
-            "q": part_number,
-            "schema": "product-offers-v0",
-            "external": True,
-            "limit": 1,
-        },
+        params=(
```
Comment (shrik450) [inline entrypoint.py:61]: That's weird. The requests docs say that a dictionary is acceptable for params, and the typeshed types allow anything that is iterable as pairs. Why is this failing typechecking?

Comment (jtran) [reply entrypoint.py:61]: I don't know why it's failing. Yes, it should work. But the types used are quite complicated.

**Review by shrik450 — APPROVED** [review]

File: entrypoint.py (hunk 1 above)
Comment (jtran) [reply entrypoint.py]: I changed to an alternate way that allows use of a dictionary. What do you think?

**Review by shrik450 — APPROVED** [review]
Maybe we should report this as a bug to typeshed? Either way, I’m glad it’s a duct instead of a tuple of tuples.

**Pull request merged**